### PR TITLE
updated CHANGELOG.md

### PR DESCRIPTION
**Pull request by jackmcintosh**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+# 1.2.0 
+Pull Request [#5] (https://github.com/BLOCKvIO/web-sdk/pull/5)
+## Added:
+Chat api endpoint
+
+Delete vAtom endpoint
+
+EventEmitter class
+
+Websocket Documentation
```
Comment (cjmconie) [inline CHANGELOG.md]: Websocket > "Web socket"

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+Delete vAtom endpoint
+
+EventEmitter class
+
+Websocket Documentation 
+ - Pull Request [#6] (https://github.com/BLOCKvIO/web-sdk/pull/6)
+
+## Updated:
+Updated Readme to include Websockets and Discover Query
```
Comment (cjmconie) [inline CHANGELOG.md]: "Websocket" > "Web socket" two words.

File: CHANGELOG.md (hunk 1 above)
Comment (jackmcintosh) [reply CHANGELOG.md]: fixed in e05aac3

File: CHANGELOG.md (hunk 2 above)
Comment (jackmcintosh) [reply CHANGELOG.md]: fixed in e05aac3

**Pull request merged**